src/main/java/spotify/api/enums/AlbumType.java

```diff
@@ -8,6 +17,11 @@
 
     private final String value;
 
+    /**
+     * Set the type of the album: album, single, appears_on, or compilation.
```

Comment:
This is a constructor so it would make more sense to write something like: `Constructs an enum value accepting a string argument whose value is the string representation of itself`